cgo/lapack.go

```diff
@@ -100,9 +156,6 @@ func (impl Implementation) Dpotrf(ul blas.Uplo, n int, a []float64, lda int) (ok
 //
 // Work is temporary storage of length at least n and this function will panic otherwise.
 func (impl Implementation) Dgeqr2(m, n int, a []float64, lda int, tau, work []float64) {
-	// TODO(btracey): This is oriented such that columns of a are eliminated.
```

Comment:
Because nothing can be done about the implementation of this routine in C. The comment remains in the Go code (which we could do something about).